sample/src/main/java/com/google/modernstorage/sample/filesystem/FileSystemViewModel.kt

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.modernstorage.sample.filesystem
+
+import android.app.Application
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.net.Uri
+import android.os.Parcelable
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.viewModelScope
+import com.google.modernstorage.filesystem.AndroidFileSystems
+import com.google.modernstorage.filesystem.AndroidPaths
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import kotlinx.parcelize.Parcelize
+import java.nio.file.Files
+import java.nio.file.Path
+import java.util.stream.Collectors
+
+const val CURRENT_DOCUMENT_KEY = "currentDocument"
+
+class FileSystemViewModel(
```

Comment:
Comment about this class (and probably calling out that it's only 26+, along with a @RequiresApi(Build.VERSION_CODES.O) annotation.